src/main/scala/Core/RegisterFile.scala

```diff
@@ -61,19 +61,22 @@ class RegisterFile(val threads: Int, val readPorts: Int = 2, val writePorts: Int
       0.U
     )
 
+    // FIXME: Shaokai: Looks suspicious.
```

Comment:
?

Reply:
Sorry, this legacy comment was left here when I looked at the original RegisterFile code. Thanks for catching this...